web/src/message_edit.ts

```diff
@@ -508,7 +508,10 @@ function edit_message($row: JQuery, raw_content: string): void {
     currently_editing_messages.set(message.id, $message_edit_content);
     message_lists.current.show_edit_message($row, $form);
 
-    $message_edit_content.on("keydown", (e) => {
+    // Attach event handlers to `form` instead of `textarea` to allow
+    // typeahead to call stopPropagation if it can handle the event
+    // and prevent the form from submitting.
+    $form.on("keydown", (e) => {
```

Comment:
It looks like this bug was introduced by @evykassirer in dc938768ca739cdd8c86f6e1f4b7064a5cf1b0c5. Changing selector scopes is probably not something we should be doing in a commit described as doing something else.

Reply:
Thanks for calling my attention to this! Sorry about that